tests/logging_tests/tests.py

```diff
@@ -105,6 +106,47 @@ def test_django_logger_debug(self):
             self.logger.debug('debug')
             self.assertEqual(self.logger_output.getvalue(), '')
 
+    def test_page_found_no_warning(self):
+        with self.settings(DEBUG=True):
```

Comment:
I'd decorate the test method with `@override_settings(DEBUG=True)` instead to avoid extra indentation.